README.md

```diff
@@ -1,20 +1,88 @@
 # Flow Data Provisioning Service
 
+The Flow Data Provisioning Service aims at providing a scalable and efficient way to access the execution state of past
+and live sporks, as well as implementing the Rosetta API standard.
+
+Past sporks are loaded from the filesystem using Flow's Write-Ahead Log to replay the changes to all registries, and
+updates are pushed directly to the DPS by the execution nodes in order to handle live networks.
+
+The Flow DPS maintains different specialized indexes for different purposes. One index is used for accessing the entire
+execution state at any given height, while the other is used to follow the history of a specific registry through time.
```

Comment:
`while another` (there are more than two, if we are precise)
`a specific Ledger register over time`

Reply:
```suggestion
execution state at any given height, while another is used to follow the history of a specific Ledger register over time.
```